sweep/store_mock.go

```diff
@@ -2,54 +2,58 @@ package sweep
 
 import (
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/stretchr/testify/mock"
 )
 
 // MockSweeperStore is a mock implementation of sweeper store. This type is
 // exported, because it is currently used in nursery tests too.
 type MockSweeperStore struct {
-	ourTxes map[chainhash.Hash]struct{}
+	mock.Mock
 }
 
 // NewMockSweeperStore returns a new instance.
 func NewMockSweeperStore() *MockSweeperStore {
```

Comment:
Are we using this anywhere in the codebase, could not find anything ?

Reply:
yeah it's not used anywhere, only being initialized in `createSweeperTestContext` but I believe the actual methods are not being hit in any of the tests. It seems to me that we only have it there, and write a unit test for this mock store...typical "do you write unit test to test your unit test" conundrum.